nautobot/docs/user-guide/administration/installation/install_system.md

```diff
@@ -40,406 +40,452 @@ You must select either MySQL (MariaDB is **not** supported) or PostgreSQL. Postg
+            affiliates. Other names may be trademarks of their respective
+            owners.
+
+            Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+            mysql> CREATE DATABASE nautobot;
+            Query OK, 1 row affected (0.00 sec)
+
+            mysql> CREATE USER 'nautobot'@'localhost' IDENTIFIED BY 'insecure_password';
+            Query OK, 0 rows affected (0.00 sec)
+
+            mysql> GRANT ALL ON nautobot.* TO 'nautobot'@'localhost';
+            Query OK, 0 rows affected (0.00 sec)
+
+            mysql> \q
+            Bye
+            ```
+
+        ### Verify MySQL Service Status
+
+        You can verify that authentication works issuing the following command and providing the configured password.
+
+        If successful, you will enter a `mysql>` prompt. Type `status` to confirm your connection, or type `\q` to exit.
+
+        !!! note
+            Replace `localhost` below with your database server if using a remote database.
+
+        ```no-highlight
+        mysql --user nautobot --password --host localhost nautobot
+        ```
+
+        ??? example "Example MySQL Verification."
+
+            ```no-highlight
+            Enter password:
+            Welcome to the MySQL monitor.  Commands end with ; or \g.
+            Your MySQL connection id is 10
+            Server version: 8.0.21 Source distribution
+
+            Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+            Oracle is a registered trademark of Oracle Corporation and/or its
+            affiliates. Other names may be trademarks of their respective
+            owners.
+
+            Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+            mysql> status
+            --------------
+            mysql  Ver 8.0.21 for Linux on x86_64 (Source distribution)
+
+            Connection id:          10
+            Current database:       nautobot
+            Current user:           nautobot@localhost
+            SSL:                    Not in use
+            Current pager:          stdout
+            Using outfile:          ''
+            Using delimiter:        ;
+            Server version:         8.0.21 Source distribution
+            Protocol version:       10
+            Connection:             Localhost via UNIX socket
+            Server characterset:    utf8mb4
+            Db     characterset:    utf8mb4
+            Client characterset:    utf8mb4
+            Conn.  characterset:    utf8mb4
+            UNIX socket:            /var/lib/mysql/mysql.sock
+            Binary data as:         Hexadecimal
+            Uptime:                 4 min 12 sec
+
+            Threads: 2  Questions: 12  Slow queries: 0  Opens: 151  Flush tables: 3  Open tables: 69  Queries per second avg: 0.047
+            --------------
+
+            mysql> \q
+            Bye
+            ```
 
 ### Troubleshooting
```

Comment:
This section is weirdly placed because you're not going to encounter the given Django output at this stage in the process. We should really relocate it - not blocking for this PR though.